docs/logs.asciidoc

```diff
@@ -3,8 +3,20 @@ NOTE: For the best reading experience,
 please view this documentation at https://www.elastic.co/guide/en/apm/agent/dotnet[elastic.co]
 endif::[]
 
-[[log-correlation]]
-== Log correlation
+[[logs]]
+== Logs
+
+Elastic .NET Agent provides <<log-correlation-ids>>.
+The agent will automaticaly inject correlation IDs that allow navigation between logs, traces and services.
+
+This features is part of {observability-guide}/application-logs.html[Application log ingestion strategies].
+
+The {ecs-logging-dotnet-ref}/intro.html[`ecs-logging-dotnet`] library can also be used to use the {ecs-logging-ref}/intro.html[ECS logging] format without an APM agent.
+When deployed with the .NET APM agent, the agent will provide <<log-correlation-ids,log correlation>> IDs.
```

Comment:
```suggestion
ECS .NET logging will always provide <<log-correlation-ids,log correlation>> IDs through https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.activity?view=net-7.0[`System.Diagnostics.Activity`]. When paired with the APM Agent it will pick up the appropriate service correlation features too.
```